src/Supplier/AddSupplierTittle.tsx

```diff
@@ -0,0 +1,11 @@
+import { Box } from '@chakra-ui/react';
+
+function AddSupplierTittle() {
```

Comment:
It is `title` with one `t` inside.

Reply:
You have right, i have changed this

Reply:
You didn't. It looks the same.
![image](https://user-images.githubusercontent.com/48626088/111861655-574d3a00-8950-11eb-87e3-b02b0bd01d5c.png)